### PR TITLE
Added support for DraftOrders

### LINKED ISSUE #6: Draft Order support
_From @baorv on July 17, 2018 10:45_

Should support Draft Order. 
[https://help.shopify.com/en/api/reference/orders/draftorder](https://help.shopify.com/en/api/reference/orders/draftorder)

_Copied from original issue: getconversio/go-shopify#103_

### PR DESCRIPTION
**Pull request by ckeyes88**
closes #6

### CODE REVIEW

File: draft_order.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+	Name            string           `json:"name,omitempty"`
+	Customer        *Customer        `json:"customer,omitempty"`
+	ShippingAddress *Address         `json:"shipping_address,omitempty"`
+	BillingAddress  *Address         `json:"billing_address,omitempty"`
+	Note            string           `json:"note,omitempty"`
+	NoteAttributes  []NoteAttribute  `json:"note_attribute,omitempty"`
+	Email           string           `json:"email,omitempty"`
+	Currency        string           `json:"currency,omitempty"`
+	InvoiceSentAt   *time.Time       `json:"invoice_sent_at,omitempty"`
+	InvoiceURL      string           `json:"invoice_sent_at,omitempty"`
```
Comment (markstgodard) [inline draft_order.go]: InvoiceURL tag should be "invoice_url",  looks like it is repeating "invoice_sent_at"

File: draft_order.go [hunk 2]
```diff
@@ -0,0 +1,215 @@
+package goshopify
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/shopspring/decimal"
+)
+
+const draftOrdersBasePath = "admin/draft_orders"
```
Comment (markstgodard) [inline draft_order.go]: Minor shorter hand simplification:
```go
const (
	draftOrdersBasePath     = "admin/draft_orders"
	draftOrdersResourceName = "draft_orders"
)
```

**Comment by markstgodard** [comment]
Hey thanks for the PR, will take a look and gets  some 👀 on it!
Cheers

**Comment by ckeyes88** [comment]
@markstgodard Thanks for looking it over, changes are up!

**Comment by markstgodard** [comment]
LGTM

**Pull request merged**